package day11Assignment;

public class ThrowThrows4 {

	public static void main(String[] args) throws ArithmeticException{
		int a=10;
		for(int i=3;i>=0;i--)
		{
				
				System.out.println( a/i);
		}
			System.out.println("between try and catch");
		

	}

}
